payments/credit-account/redirect.md

```diff
@@ -0,0 +1,386 @@
+---
+title: Swedbank Pay Payments Credit Account Redirect
+sidebar:
+  navigation:
+  - title: Credit Account Payments
+    items:
+    - url: /payments/credit-account/
+      title: Introduction
+    - url: /payments/credit-account/redirect
+      title: Redirect
+    - url: /payments/credit-account/after-payment
+      title: After Payment
+    - url: /payments/credit-account/other-features
+      title: Other Features
+---
+
+{% include alert-review-section.md %}
+
+{% include jumbotron.html body="Swedbank Pay Credit Account is an online payment
+instrument allowing payers to split a purchase into several payments.
+The basic redirect purchase scenario is the simplest and most common way to
+implement Credit Account Payments." %}
+
+{% include alert.html type="info"
+                      icon="info"
+                      body="Swedbank Pay Credit Account is only available as a
+                      payment instrument for swedish merchants." %}
+
+## Introduction
+
+Redirect is the integration that lets Swedbank Pay handle the payments, while
+you handle your core activities. When ready to pay, the consumer will be
+redirected to a secure Swedbank Pay hosted site. Finally, the consumer will be
+redirected back to your website after the payment process.
+
+* When properly set up in your merchant/webshop site and the payer starts the
+  purchase process, you need to make a `POST` request towards Swedbank Pay with
+  your Purchase information. This will generate a payment object with a unique
+  `paymentID`. You will receive a **redirect URL** to a Swedbank Pay payment
+  page.
+* You need to redirect the payer's browser to that specified URL so that the
+  payer can enter the payment details in a secure Swedbank Pay environment.
+* Swedbank Pay will redirect the payer's browser to - one of two specified URLs,
+  depending on whether the payment session is followed through completely or
+  cancelled beforehand. Please note that both a successful and rejected payment
+  reach completion, in contrast to a cancelled payment.
+* When you detect that the payer reach your `completeUrl` , you need to do a
+  `GET` request to receive the state of the transaction, containing the
+  `paymentID` generated in the first step, to receive the state of the
+  transaction.
+
+This is how the payment window might look like:
+
+**Ha med eksempel av screenshot her**
```

Comment:
Fix screenshot pl0x.

Reply:
waiting for Are to complete his work and give me screenshots